pkg/logstructured/logstructured.go

```diff
@@ -33,7 +33,12 @@ func (l *LogStructured) Start(ctx context.Context) error {
 	if err := l.log.Start(ctx); err != nil {
 		return err
 	}
-	l.Create(ctx, "/registry/health", []byte(`{"health":"true"}`), 0)
+	// See https://github.com/kubernetes/kubernetes/blob/442a69c3bdf6fe8e525b05887e57d89db1e2f3a5/staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go#L97
+	if _, err := l.Create(ctx, "/registry/health", []byte(`{"health":"true"}`), 0); err != nil {
+		if err != server.ErrKeyExists {
+			logrus.Errorf("Failed to create health check key: %v", err)
```

Comment:
Do we want to return here or keep processing?

Reply:
We previously just ignored the return; I added this because one of the linters complained about it but it doesn't seem to be truly critical? If something is actually wrong with the database it'll fail immediately afterwards anyways.